core-bundle/src/EventListener/Undo/UndoDescriptionListener.php

```diff
@@ -69,10 +65,11 @@ static function ($field) use ($row) {
         return vsprintf($format, $values);
     }
 
-    private function getFallbackDescription(array $row): ?string
+    private function getFallbackDescription(string $table, array $row, array $options = []): ?string
     {
         foreach (['title', 'username', 'email', 'name', 'headline'] as $key) {
             if (!empty($row[$key])) {
+                // TODO: Handle field according to `inputType` and `eval` settings in DCA
```

Comment:
Right now, the listeners only take the field as-is, with no handling whatsoever. Is there a helper class, that takes care of handling the values according to the DCA already?